README.md

```diff
@@ -107,6 +107,23 @@ Docker to work with the helper.
 To build and install the Amazon ECR Docker Credential Helper, we suggest Go
 1.9+, `git` and `make` installed on your system.
 
+If you just installed Go, make sure you also have added it to your PATH or 
+Environment Vars (Windows). For example:
+
+```
+$ export GOPATH=$HOME/gocode
+$ export PATH=$PATH:$GOPATH/bin
+```
+
+Or in Windows:
+
+```
+<your existing PATH definitions>;C:\Users\<username>\gocode\bin\
```

Comment:
Same as above, this could be `%USERPROFILE%\go\bin`.  And I think you might want to add `setx GOPATH %USERPROFILE%\go` to be consistent with the advice for Linux.

(Windows commands are documented in [the Go wiki](https://github.com/golang/go/wiki/SettingGOPATH))

Reply:
Fixed.